mayastor/src/bdev/dev/aio.rs

```diff
@@ -55,8 +55,19 @@ impl TryFrom<&Url> for Aio {
             },
         )?;
 
-        if let Some(keys) = uri::keys(parameters) {
-            warn!("ignored parameters: {}", keys);
+        if !parameters.is_empty() {
```

Comment:
Shall we put this in a function instead of rewriting it?

Reply:
Agree. :)